src/StatusQ/Popups/statusModal/StatusImageWithTitle.qml

```diff
@@ -12,8 +12,10 @@ Row {
     property int titleElide: Text.ElideRight
     property int subTitleElide: Text.ElideRight
     property bool editable: false
+    property bool editAvatar: false
```

Comment:
Another nitpick: how about naming this: `headerImageEditable` instead across all boards?

Reply:
no problem :)